src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days > 2 && days < 7) {
+    return days * 40 - 20;
+  }
+
+  if (days > 6) {
+    return days * 40 - 50;
+  }
 }
 
 module.exports = calculateRentalCost;
```

Comment:
wow . what does it  do ; )?

Reply:
when there are at least 7 days it returns days * 40(because that's the price for a day) and -50 because that's a discount for 7 days or more

Reply:
I asked about module.exports. Never mind, sorry i didn't approve immediately as everything was fine.